fiftyone/core/odm/database.py

```diff
@@ -1484,6 +1484,98 @@ def delete_runs(name, dry_run=False):
     )
 
 
+def get_indexed_values(
+    dataset, field_or_index, pipeline=None, values_only=False
+):
+    """Returns the values of the field for all samples in the given collection that are covered by the index.
```

Comment:
Please respect the 79 character line limit from the [style guide](https://github.com/voxel51/fiftyone/blob/develop/STYLE_GUIDE.md)!

The pre-commit hooks enforce it automatically on code, but you must manually enforce it in comments/docstrings.